config/config_files/setupLocalhost.mjs

```diff
@@ -1,9 +1,10 @@
-import ora from 'ora';
+/* eslint-disable no-console */
```

Comment:
Is this needed? Same with the eslint disable further down

Reply:
It just suppresses the lint warnings, gives warning if not, so thought it would be fine to just ignore the warnings.